atlasdb-ete-tests/src/main/java/com/palantir/atlasdb/blob/BlobSchema.java

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright 2018 Palantir Technologies, Inc. All rights reserved.
+ *
+ * Licensed under the BSD-3 License (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://opensource.org/licenses/BSD-3-Clause
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.palantir.atlasdb.blob;
+
+import java.io.File;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.palantir.atlasdb.keyvalue.api.Namespace;
+import com.palantir.atlasdb.protos.generated.TableMetadataPersistence;
+import com.palantir.atlasdb.schema.AtlasSchema;
+import com.palantir.atlasdb.schema.stream.StreamStoreDefinitionBuilder;
+import com.palantir.atlasdb.table.description.Schema;
+import com.palantir.atlasdb.table.description.TableDefinition;
+import com.palantir.atlasdb.table.description.ValueType;
+import com.palantir.logsafe.UnsafeArg;
+
+public class BlobSchema implements AtlasSchema {
+    private static final Logger log = LoggerFactory.getLogger(BlobSchema.class);
+
+    private static final Namespace BLOB_NAMESPACE = Namespace.create("blob");
+    private static final Schema GENERATED_SCHEMA = generateSchema();
+
+    @Override
+    public Namespace getNamespace() {
+        return BLOB_NAMESPACE;
+    }
+
+    @Override
+    public Schema getLatestSchema() {
+        return GENERATED_SCHEMA;
+    }
+
+    public static Schema getSchema() {
+        return GENERATED_SCHEMA;
+    }
+
+    private static Schema generateSchema() {
+        Schema schema = new Schema(
+                BlobSchema.class.getSimpleName(),
+                BlobSchema.class.getPackage().getName() + ".generated",
+                BLOB_NAMESPACE);
+
+        schema.addStreamStoreDefinition(
+                new StreamStoreDefinitionBuilder("data", "Data", ValueType.VAR_LONG)
+                .hashRowComponents()
+                        .tableNameLogSafety(TableMetadataPersistence.LogSafety.SAFE)
```

Comment:
nit: whitespace looks out-of-whack here

Reply:
Ack, will fix this. Not sure what happened here!